classes/class0/README.md

```diff
@@ -0,0 +1,62 @@
+#Getting Ready for Olin.js
+
+##Ubuntu
+If you need help install Ubuntu check out this [guide](https://gist.github.com/benkahle/892fdb2531c5f388296a) written specifically for Olin laptops. It also includes some other useful packages and tips if you are new to Ubuntu.
+
+##Node
+We will install node from a package repository which will give us automatic updates:
+* `curl -sL https://deb.nodesource.com/setup | sudo bash -` (install curl with `sudo apt-get install curl` if needed)
+* `sudo apt-get install -y nodejs`
+* `sudo apt-get install build-essential` (needed for some npm packages)
+* To test: `node --version` should return v0.10.35 and `npm --version` should return 2.1.17
+* If your npm is not up-to-date run `sudo npm install -g npm`
+
+##Github
+We are going to be using git for homework submission and many other things in this course. The way this will work is that you will fork this git repo into your own account and then you will push your homework to that repository when you finish it.
+
+To start out you will need a Github account. Either log in or create a new one at https://github.com.
+
+If you don't have git on your computer already, it can be installed with:
+```sh
+sudo apt-get install git
+```
+
+Lastly, ensure you are setup to use SSH to communicate with Github. SSH is a communication protocol that ensures secure transfer of information while preventing you from needing to enter your username and password to push. Follow [this guide]((https://help.github.com/articles/generating-ssh-keys/) if you need any help. 
+
+### 1. Forking
+
+"Forking" is to copy a repository. It's termed "forking" because you copy the entire history of the project, but from here on out, any changes you make to your new repository are distinct from those you make in the original repository. It's totally fine to keep two repositories ongoing with the same history, it's just data! 
```

Comment:
I'd change the first sentence to this:

>"Forking" makes your own personal copy of a repository. The term is "forking" because you copy the entire history of the project, but from here on out, any changes you make to your new repository are distinct from those made in the original repository.